content/applications/sales/sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -1,51 +1,52 @@
-===================================
-Get a signature to confirm an order
-===================================
+=========================================
+Online signatures for order confirmations
+=========================================
 
-Online signatures are like electronic "fingerprints". By using them on Odoo, you will get
-automatic orders confirmation. You and your customers will save a lot of time by using this
-feature compared to a traditional process.
+The Odoo *Sales* application provides customers with the ability to confirm orders via an online
+signature directly on the sales order. Once the sales order is electronically signed by the
+customer, the salesperson attached to the sales order is automatically notified that the order is
+confirmed.
 
-Activate online signature
-=========================
+Activate online signatures
+==========================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Signature**
-feature.
+To activate the *online signature* feature, go to :menuselection:`Sales app --> Configuration -->
+Settings`, scroll to the :guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online
+Signature` feature by checking the box beside it, and clicking the :guilabel:`Save` button in the
+top-left corner.
 
-.. image:: get_signature_to_validate/signature_1.png
+.. image:: get_signature_to_validate/signature-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable online signature on Odoo Sales?
+   :alt: How to enable online signature in Odoo Sales settings.
 
 .. note::
-   If you are familiar with this documentation: :doc:`quote_template`, you can activate or not the
-   **Online Signature** feature for each quotation template you use, under their confirmation tab.
-   Example:
+   The :guilabel:`Online Signature` feature for each quotation template is found under the
```

Comment:
Changing the sentence structure slightly to mirror the following sentence and draw the comparison more clearly.
```suggestion
   On quotation templates, the :guilabel:`Online Signature` feature is located under the
```